mindmeld/rasaconverter.py

```diff
@@ -0,0 +1,535 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains the Rasacoverter class used to convert Rasa projects
+into Mindmeld projects"""
+
+from keyword import iskeyword
+import re, os, yaml, copy
+from converter import Converter
+
+
+class Rasaconverter(Converter):
+
+    def __init__(self, rasa_project_directory, mindmeld_project_directory):
+        self.rasa_project_directory = rasa_project_directory
+        self.mindmeld_project_directory = mindmeld_project_directory
+
+    @staticmethod
+    def __create_intents_directories(mindmeld_project_directory, intents):
+        for intent in intents:
+            Converter.create_directory(mindmeld_project_directory + "/domains/default/" + intent) 
+
+    @staticmethod
+    def __create_entities_directories(mindmeld_project_directory, entities):
+        for entity in entities:
+            Converter.create_directory(mindmeld_project_directory + "/entities/" + entity)
+            print(mindmeld_project_directory + "/entities/" + entity)
```

Comment:
remove print

Reply:
ah, nvm, I thought this was a debugging print

Reply:
Yes, I think that might be a a debugging print. I think it's ok if I remove it.